app/forms/candidate_interface/english_gcse_grade_form.rb

```diff
@@ -22,7 +22,8 @@ class EnglishGcseGradeForm
                   :grade_english_studies_double,
                   :other_english_gcse,
                   :other_english_gcse_name,
-                  :grade_other_english_gcse
+                  :grade_other_english_gcse,
+                  :award_year
```

Comment:
Do we need award year here anymore? It is not referenced anywhere else in that file.

Reply:
`award_year` had to be added to the form so that the `next_gcse_step` method in the controller could determine the next route. 
https://github.com/DFE-Digital/apply-for-teacher-training/pull/3466/files#diff-7a31994bcda7a7f3914b9d5cf92495970d8b7b2c2ff52876b1eeb6c148787924R56
Previously this logic was happening in the concern. Now each grade controller needs to handle this separately has they are using different forms.

Reply:
Got it, thanks for explaining Toby 👍